### PR TITLE
[PCFG-136] Ordering

### PR DESCRIPTION
**Pull request by aomegax**
Depends on #49 

#### List of Changes
- Added asc/desc ordering option to all section

#### Screenshots (if appropriate):
image

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/components/Ordering.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {FaSort, FaSortDown, FaSortUp} from "react-icons/fa";
+
+interface IProps {
+    currentOrderBy: string;
+    currentOrdering: string;
+    orderBy: string;
+    ordering: string;
```
Comment (jacopocarlini) [inline src/components/Ordering.tsx:8]: is this used?

File: src/pages/brokersPSP/BrokersPSP.tsx [hunk 2]
```diff
@@ -207,8 +234,14 @@ export default class BrokersPSP extends React.Component<IProps, IState> {
                                     <Table hover responsive size="sm">
                                         <thead>
                                         <tr>
-                                            <th className="fixed-td-width">Intermediario PSP</th>
-                                            <th className="fixed-td-width">Codice</th>
+                                            <th className="fixed-td-width">
+                                                <Ordering currentOrderBy={this.state.order.by} currentOrdering={this.state.order.ing} orderBy={"CODE"} ordering={"DESC"} handleOrder={this.handleOrder} />
```
Comment (jacopocarlini) [inline src/pages/brokersPSP/BrokersPSP.tsx:238]: `ordering={"DESC"}` is the initial ordering?

File: src/components/Ordering.tsx (hunk 1 above)
Comment (aomegax) [reply src/components/Ordering.tsx:8]: yes, they are used

**Review by jacopocarlini — APPROVED** [review]
🚀

**Pull request merged**